tests/components/Search.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```

Comment:
This test doesn't run because you told mocha to look for `*.test.js` which doesn't work for this `.jsx` file. You need to change that to `*.test.js*`